itools-packager/src/main/resources/itools.bat

```diff
@@ -48,17 +48,18 @@ if "%~1"=="" ( goto done ) else (
 :done
 
 set options=
-if "%powsybl_config_dirs%" == "" ( set powsybl_config_dirs=%installDir%\etc:%HOMEDRIVE%%HOMEPATH%\.itools)
+if "%powsybl_config_dirs%" == "" ( set powsybl_config_dirs=%installDir%\etc;%HOMEDRIVE%%HOMEPATH%\.itools)
 set options=%options% -Dpowsybl.config.dirs="%powsybl_config_dirs%"
 if not "%powsybl_config_name%" == "" ( set options=%options% -Dpowsybl.config.name=%powsybl_config_name%)
 
 set options=%options% -Dlogback.configurationFile="
-if exist %powsybl_config_dirs%\logback-itools.xml (
-  set options=%options%%powsybl_config_dirs%
-) else (
-  set options=%options%%installDir%\etc
+set logFile=%installDir%\etc\logback-itools.xml
+for %%G in ("%powsybl_config_dirs:;=";"%") do (
+    if exist %%G\logback-itools.xml (
+        set logFile=%%G\logback-itools.xml
+    )
 )
-set options=%options%\logback-itools.xml"
+set options=%options%%logFile%"
```

Comment:
Add quotes to handle spaces in paths

Reply:
The answer was a bit troublesome but the last commit seems to handle that case (tested on Windows)